### PR TITLE
Quest changes

### PR DESCRIPTION
**Pull request by Pxx500**
Repositioned coil quests to clearly show which ones are optional, enabled dependency lines for lategame coils
image

Edited EBF quest to accomodate new voiding mechanics
image

Thermosink Radiator is now correctly dependent on spacetime quest, not Finally, you are here quest, also moved slightly
image

Slow building ring quest has been deleted just as the item
image

Platinum group metals quests no longer require non-significant outputs
image
image
image
image
image
image

New Relocator quest
image

Fixed error in telepad quest, the quest now no longer requires americium bee but "making Americium" (looks like someone assumed that quest "americium" means it's just normal americium quest https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13931)
image

New IC2 teleporter quest
image

Quests about teleporting moved to accommodate new ones
image

New Integrated Ore Factory Quest
image
Located:
image
image

Infinite fluid drilling rig quest
image

New P2P interface quests
image
image
image

New Fluid quantum storage and digital singularity quests (the descriptions are not good, comments with better ones are appreciated)
image
image

Minor tweaks to the AE tab layout
image

New Thaumometric essentia cell quest (might need some lore)
image
image

Space Elevator quest changed a little
image

Fixed massive lag in nuclear physiscs tab due to deleted GTpp items (ingots set as placeholder icons)
image

Changes to the lutetium quests (also deleted one duplicate quest)
image

Removed mention of HTGR in power side of the fission quest branch

Realligned quests and fixed dependencies
image

Changed to OR logic
image

New UIV and UMV capacitor quests
image
image

New SoC circuit quests
image
image
image

Updated TFT quest
image

New quest about visual prospecting in stone age tab
image
image

BIOS quest no longer consumes the eeprom
image

Oil quest changes
image
image
image
image
image
image

Renamed methanol and ethanol quests from "ethanols" to "alcohols"

### CODE REVIEW

File: config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
...
...id you can recover. Bart\u0027s Platinum Series recipes output fluids on the BOTTOM, so if you want to do Platinum Stuff later, remember that. If you want to void fluids anyway, just enable fluid voiding in the controller.\n\nMoron always says - Make Muffler Moint Mup. Or point up, anyways. Before the launch, don\u0027t forget to fix maintenance issues and put a programmed circuit if necessary.\n\nIf you want a really fancy EBF, you can add more input fluid hatches and store different fluids in each one.\n\nThe EBF also has a heat adjustment of 100K for each power tier above or below MV. You can only make LV energy hatches right now, but installing two LV energy hatches will overclock the EBF to MV power, eliminating the 100K penalty and setting the initial heat to 1801K.\n\nAs you upgrade your EBF over the course of the game, this heat adjustment will increase. Two MV energy hatches (overclocking to HV) will add +100K, two HV energy hatches (overclocking to EV) will give +200K, and so on. Coils only set the base value. This adjustment does not apply to the Volcanus.\n\nIf the pollution gets really bad, check in GT++ Multiblocks for a pollution scrubber, or upgrade your Muffler.\n"
```
Comment (bombcar) [inline config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json:34]: "You have to put an output"

I would reword it slightly so that it is more like:

 You have to put an output fluid hatch to the top layer of the EBF to collect it, or enable fluid voiding, otherwise the EBF won\u0027t start.

and remove the later line. Does EBF ever make good fluids?

Comment (Pxx500) [reply config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json:34]: I never used those fluids, but I can see why someone would want to recapture them (sulfur dioxide for example for sulfuric acid).

File: config/betterquesting/DefaultQuests/Quests/PowerfulNuclearP-AAAAAAAAAAAAAAAAAAAAGg==/FasterLutetiumEx-AAAAAAAAAAAAAAAAAAAMbA==.json [hunk 2]
```diff
@@ -37,49 +29,13 @@
...
+      "desc:8": "This quest is late LuV.\n\nYou might have read before that Lutetium is necessary as an input for fusion crafting. In a tier 2 fusion reactor, Lutetium + Chrome \u003d Americium, a required material for many required crafts. Lutetium is a material that is obtained exclusively from breeding, so you\u0027re in the right place to find ways to get it.\n\nThe very first method, EV\u0027s IC2 Thorium nukes, is the cheapest, but also the slowest. Each fuel rod takes 50000 seconds to deplete, almost 14 hours, with half of a Lutetium Dust per depleted rod. Highly stackable designs, such as the 2-chamber 11 quad rod one, will grant you 22 dust per cycle. After building 10 of those, the rate is still 16 dust per hour. This processing speed isn\u0027t good enough to keep progressing forever, and there is no need to make more towers of breeder nukes.\n\nYour 3 main options are:\n\n1) HTGR - fastest but also quite expensive multiblock reactor, unfortunately it requires purified thorium232 you you will need to purify it first\n2) Lutetium bees - if you started bees getting it should be quite easy\n3) Spamming IC2 nuclear reactors"
```
Comment (bombcar) [inline config/betterquesting/DefaultQuests/Quests/PowerfulNuclearP-AAAAAAAAAAAAAAAAAAAAGg==/FasterLutetiumEx-AAAAAAAAAAAAAAAAAAAMbA==.json:32]: the part where it talks about "thorium232" has you you

File: config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json [hunk 3]
```diff
@@ -1,76 +1,62 @@
...
       "questLogic:8": "AND",
       "repeat_relative:1": 1,
-      "name:8": "So You Want to Go Slower...Wait, What?",
-      "isGlobal:1": 0,
+      "name:8": "Teleporter",
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Sometimes you need to go slow so you don\u0027t fall off the top of your spire. Or you don\u0027t want to accidentally sprint in your fields and trample your crops. Maybe you don\u0027t want to have to hold sneak all the time. This bauble will weigh you down - keeping you from accidentally sprinting, and can be activated to enable/disable sneaking."
+      "desc:8": "IndustrialCraft2 Teleporter allows for teleportation between 2 connected teleporters.\nRequires energy storing block right next to it due to the massive power demand.\n\n\n§3Energy required is determined by the distance between the teleporters and amount of items in the inventory.\nThe teleporter is quite buggy according to the offcial wiki."
```
Comment (bombcar) [inline config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json:32]: official

File: config/betterquesting/DefaultQuests/Quests/PowerfulNuclearP-AAAAAAAAAAAAAAAAAAAAGg==/FasterLutetiumEx-AAAAAAAAAAAAAAAAAAAMbA==.json (hunk 2 above)
Comment (Pxx500) [reply config/betterquesting/DefaultQuests/Quests/PowerfulNuclearP-AAAAAAAAAAAAAAAAAAAAGg==/FasterLutetiumEx-AAAAAAAAAAAAAAAAAAAMbA==.json]: done

File: config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json (hunk 3 above)
Comment (Pxx500) [reply config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json]: done

File: config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json [hunk 4]
```diff
@@ -347,7 +347,7 @@
     "43:10": {
       "questIDHigh:4": 0,
       "sizeX:3": 24,
-      "x:3": 252,
+      "x:3": 240,
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json:350]: the quests are assymmetric now. don't think that's better

Comment (Pxx500) [reply config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json:350]: please provide a screenshot of what quest is that

File: config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json (hunk 1 above)
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json]: You have put an output fluid hatch to the top layer -> You have to put an output fluid hatch on the top layer

File: config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json (hunk 4 above)
Comment (Pxx500) [reply config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json:350]: hm I assume you mean thermosink radiator quest, I can revert it but the lines will overlap, is that ok?

Comment (chochem) [reply config/betterquesting/DefaultQuests/QuestLines/EndgameGoals-AAAAAAAAAAAAAAAAAAAAHQ==.json:350]: yea

File: config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json (hunk 1 above)
Comment (chochem) [reply config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json]: outdated apparently, I went commit by commit

Comment (Pxx500) [reply config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json]: actually not outdated,, other things were changed

File: config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Relocator-8QSORTVOSOaUXC6mQurMgA==.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Relocator",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Relocator allows for teleportation to previously marked locations, even between dimensions.\n\n§3Uses quite a lot of power so GraviChestPlate or other energy sources are recommended."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Relocator-8QSORTVOSOaUXC6mQurMgA==.json:35]: 'The' at the start missing I assume. Also 'a GraviChestPlate'

File: config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Teleporter",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "IndustrialCraft2 Teleporter allows for teleportation between 2 connected teleporters.\nRequires energy storing block right next to it due to the massive power demand.\n\n\n§3Energy required is determined by the distance between the teleporters and amount of items in the inventory.\nThe teleporter is quite buggy according to the offcial wiki."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json]: This also has a bunch of articles missing. 'The IndustrialCraft2 Teleporter...', 'It requires an energy storing block...', 'The energy required ...', 'the amount of items'

Comment (Pxx500) [reply config/betterquesting/DefaultQuests/Quests/GettingAround-bAal8kZ3R8eRTwaHgbYkKA==/Teleporter-tJdwTeS6TOOFACQhNuUvEw==.json]: english is hard :fail:

File: config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/IntegratedOreFac-3hN7FykMSJSWSykZiEWvfQ==.json [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Integrated Ore Factory - All in One!",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Integrated Ore Factory allows you to process ores through multiple processes at once."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/IntegratedOreFac-3hN7FykMSJSWSykZiEWvfQ==.json:36]: also a 'The' at the start

File: config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/SomethingFromNot--1GoIXIXR029oc0Ge6sRGg==.json [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "§6§lSomething From Nothing pt 2",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "This multiblock works just as a normal drilling rig, but with the power of §5§l§oTechnology §rit won\u0027t reduce the underground vein size so the output will always be the same."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/SomethingFromNot--1GoIXIXR029oc0Ge6sRGg==.json:36]: maybe underground fluid or underground field? vein sounds like ores

File: config/betterquesting/DefaultQuests/Quests/AppliedEnergisti-AAAAAAAAAAAAAAAAAAAAIw==/UltimateParallel-_bEWg1FHSmKu9MNhlbDlwA==.json [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "§6§lUltimate Parallelization...",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Allows for parallelization of a single intefrace, you don\u0027t have to copy patterns for multiple interfaces anymore."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/AppliedEnergisti-AAAAAAAAAAAAAAAAAAAAIw==/UltimateParallel-_bEWg1FHSmKu9MNhlbDlwA==.json:32]: intefrace->interface

File: config/betterquesting/DefaultQuests/Quests/StoringandTransf-AAAAAAAAAAAAAAAAAAAAIQ==/Atlastthefinalca-vrE4rYLuSnmmtAklayd6gw==.json [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+          "id:8": "kekztech:kekztech_lapotronicenergyunit_block",
+          "Count:3": 1,
+          "Damage:2": 10,
+          "OreDict:8": ""
+        }
+      },
+      "taskID:8": "bq_standard:retrieval"
+    }
+  },
+  "rewards:9": {
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/StoringandTransf-AAAAAAAAAAAAAAAAAAAAIQ==/Atlastthefinalca-vrE4rYLuSnmmtAklayd6gw==.json]: UHV lootbags dont exist. its just a placeholder item. They arent planned to be added either.

**Review by chochem — APPROVED** [review]
thx for the fixes and all the new quests, thats massive

**Comment by chochem** [comment]
something bigger I missed in the review or why forced to draft?

**Comment by Pxx500** [comment]
> something bigger I missed in the review or why forced to draft?

I'll add a note about small ores in the steam age quest and asdd the heavy oil mention from monthly quest issues
